File: app/src/main/java/com/mindiii/lasross/model/NotificationModel.java
package com.mindiii.lasross.model;

public class NotificationModel {

    String ItemHeading;
    String ItemDetails;
    String Time;

    public NotificationModel(String itemHeading, String itemDetails, String time) {
        ItemHeading = itemHeading;
        ItemDetails = itemDetails;
        Time = time;
    }

    public String getItemHeading() {
        return ItemHeading;
    }

    public String getItemDetails() {
        return ItemDetails;
    }

    public String getTime() {
        return Time;
    }
}

File: app/src/main/java/com/mindiii/lasross/firebase_notification/model/NotificationModel.kt
package com.mindiii.lasross.firebase_notification.model

import java.io.Serializable


data class NotificationModel(var reference_id: String? = "",
                             var click_action: String? = "",
                             var message: String? = "",
                             var orderId: String? = "",
                             var tittle: String? = "") : Serializable